python/setup.py

```diff
@@ -1,7 +1,14 @@
+import logging
 from setuptools import find_packages, setup
 
-with open("README.md", "r") as fh:
-    readme_description = fh.read()
+README_FILE_PATH = "../README.md"
```

Comment:
This change was necessary since tox was creating a clean test environment that didn't incorporate items in the parent directory, like the readme.